py_zipkin/logging_helper.py

```diff
@@ -250,9 +279,9 @@ def emit(self, record):
                 " to be provided for {0} span".format(span_name)
             )
         service_name = record.msg.get('service_name', None)
-        # Presence of service_name means new client span.
+        # Presence of service_name means this is to be a new local span.
         if service_name is not None:
-            self.store_client_span(
+            self.store_local_span(
```

Comment:
Yeah, see above comment. We just set both client and server annotations due to previous display issues in the UI. It's been working for us so far.